examples/metric-pull/scaler-with-collector-pull-values.yaml

```diff
@@ -90,4 +89,3 @@ otelCollector:
         logs:
           level: DEBUG
         metrics:
-          address: 0.0.0.0:8888
```

Comment:
breaking change in the upstream collector@0.128.0 https://github.com/open-telemetry/opentelemetry-collector/issues/13259